demo/pkg/subgraphs/subgraphs.go

```diff
@@ -180,6 +180,20 @@ func New(ctx context.Context, config *Config) (*Subgraphs, error) {
 		"default": pubsub.NewNATSConnector(defaultConnection).New(ctx),
 		"my-nats": pubsub.NewNATSConnector(myNatsConnection).New(ctx),
 	}
+
+	js, err := jetstream.New(defaultConnection)
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = js.CreateStream(ctx, jetstream.StreamConfig{
```

Comment:
Is this idempotent or will it fail if the stream already exists?

Reply:
I have changed to `CreateOrUpdateStream`